apps/challenges/aws_utils.py

```diff
@@ -185,7 +186,7 @@
             "logConfiguration": {{
                 "logDriver": "awslogs",
                 "options": {{
-                    "awslogs-group": "evalai-worker-{ENV}",
+                    "awslogs-group": "challenge-pk-{challenge_pk}-workers",
```

Comment:
@KhalidRmb how will this affect existing worker logs? they'll be in separate log group right?

Reply:
Those logs would still be accessible on CloudWatch I think. The display will only work for the newly launched workers from here on.